integration-tests/vitest/vitest.spec.js

```diff
@@ -0,0 +1,139 @@
+'use strict'
+
+const { exec } = require('child_process')
+
+const getPort = require('get-port')
+const { assert } = require('chai')
+
+const {
+  createSandbox,
+  getCiVisAgentlessConfig
+} = require('../helpers')
+const { FakeCiVisIntake } = require('../ci-visibility-intake')
+const {
+  TEST_STATUS,
+  TEST_TYPE
+} = require('../../packages/dd-trace/src/plugins/util/test')
+
+// tested with 1.6.0
+const versions = ['latest']
+
+versions.forEach((version) => {
+  describe(`vitest@${version}`, () => {
+    let sandbox, cwd, receiver, childProcess
+
+    before(async function () {
+      sandbox = await createSandbox([`vitest@${version}`], true)
+      cwd = sandbox.folder
+    })
+
+    after(async () => {
+      await sandbox.remove()
+    })
+
+    beforeEach(async function () {
+      const port = await getPort()
```

Comment:
do we really need getPort() ? see: https://github.com/DataDog/dd-trace-js/pull/4429/files

Reply:
probably not 🤔 , but I don't think `getPort` has ever failed for integration tests. Is `app.listen(0)` as reliable?

Reply:
Not sure but if we use it to fix flakiness in other places it's probably more reliable yeah :shrug:

Reply:
well it might be worth trying it out yeah. I'll do the change

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/4415/commits/b29cfdb18b753d98e0b083798f20f42f815560b8